receiver/azuremonitorreceiver/testdata/expected_metrics/metrics_golden.yaml

```diff
@@ -215,8 +201,7 @@ resourceMetrics:
                     - key: azuremonitor.resource_id
                       value:
                         stringValue: resourceId2
-                  startTimeUnixNano: "1681790471654481953"
```

Comment:
Why are we losing the start time?

Reply:
Since startTimeUnixNano is already a non required field, i set them all to 0, doing this gets rid of them. I figured since we are looking to not only reduce git clutter, but also make golden files a bit more readable, it would make sense to only keep the timeUnixNano and normalize those values. Is there a reason we would want to keep them and normalize those values also?